invenio_base/wsgi.py

```diff
@@ -53,11 +55,51 @@ def wsgi_proxyfix(factory=None):
        otherwise you application is susceptible to malicious attacks.
 
     .. versionadded:: 1.0.0
+
+    .. note::
+
+       You must set ``APP_PROXY_CONFIG`` with the correct values if your
+       application is running behind a proxy, otherwise your application is
+       susceptible to malicious attacks because to trust values that came from
+       the client rather than a proxy is a security vulnerability. The
+       configuration is a dictionary which looks like this:
+
+       .. code-block:: python
+
+          PROXYFIX_CONFIG = {
+              x_for: 1,  # X-Forwarded-For header set by proxy 1 will be read
+              x_proto: 0,  # No X-Forwarded-Proto header will be read
+              x_host: 0,  # No X-Forwarded-Host header will be read
+              x_port: 0,  # No X-Forwarded-Port header will be read
+              x_prefix: 0,  # No X-Forwarded-Prefix header will be read
+          }
+
+       And it will configure which HTTP headers will be taken into account
+       by your application. For example, if you have one proxy in front of
+       your application and you want your application to use the original
+       HTTP scheme, you would set the following configuration:
+
+       .. code-block:: python
+
+          PROXYFIX_CONFIG = {
+              x_proto: 1,
+          }
+
+       Which means, that the first ``X-Forwarded-Proto`` header will be taken
+       into account.
+
+    .. versionadded:: 1.2.0
     """
     def create_wsgi(app, **kwargs):
```

Comment:
Can I suggest to rename `WERKZEUG_014` to something like `WERKZEUG_GTE_014` (greater than equal)? Or something more understandable!
It is not clear to me if 14 is included or not :) I need always to check the import above